src/style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: 96px;
+  height: 96px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars__star {
+  background: url("./images/star.svg") no-repeat center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars .stars__star {
+  margin-left: 4px;
+}
```

Comment:
Pay attention to all your task and tests. If we try to use left margin we should to change it for all stars. Now in test your last star in row doesn't have margin. If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review. 
Deploy your solution after every changes. Looks like tests don't match the code 
```suggestion
.stars__star:not(:first-child) {
  margin-left: 4px;
}
```

![image](https://github.com/mate-academy/layout_stars/assets/70009205/1489a536-507c-46de-8386-2598bc144672)